src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.ts

```diff
@@ -69,6 +70,8 @@ export class SavedScenariosComponent implements OnInit {
       .getScenariosForPlan(this.plan?.id!)
       .pipe(take(1))
       .subscribe((scenarios) => {
+        this.totalScenarios = scenarios.length;
+
         this.scenariosForUser = this.showOnlyMyScenarios
           ? scenarios.filter((s) => s.user === this.user$.value?.id)
           : scenarios;
```

Comment:
I realized that if we try to filter inside of the existing filters for active/archived, and a user doesn't have any scenarios, we'll still end up with an empty state, because the counts for activeScenarios and archiveScenarios will still be zero. So I went with this approach of creating a `totalScenarios` count.